packages/react/src/patterns/sub-patterns/ContentBlock/ContentBlock.js

```diff
@@ -18,7 +18,16 @@ import { settings } from 'carbon-components';
 
 const { stablePrefix } = ddsSettings;
 const { prefix } = settings;
-
+/**
+ *
+ * @param {string} type returns inverse or default
+ * @returns {string} theme classname
+ */
+function themeClassname(type) {
```

Comment:
minor thing, let's make this a private function (`_themeClassname`) and add the `@private` to the JSDocs.